packages/chevrotain/api.d.ts

```diff
@@ -1823,13 +1833,13 @@ export interface DSLMethodOptsWithErr<T> extends DSLMethodOpts<T> {
     ERR_MSG?: string
 }
 
-export interface OrMethodOpts {
+export interface OrMethodOpts<T> {
```

Comment:
is adding a generic type augment here a breaking change?

Reply:
Would using generic parameter defaults make it none breaking?
- https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#generic-parameter-defaults